src/renderers/shared/stack/reconciler/ReactCompositeComponent.js

```diff
@@ -362,9 +338,20 @@ var ReactCompositeComponentMixin = {
       markup = this.performInitialMount(renderedElement, hostParent, hostContainerInfo, transaction, context);
     }
 
+    var debugID = 0;
+    if (__DEV__) {
+      debugID = this._debugID;
+    }
+
     if (inst.componentDidMount) {
-      if (__DEV__) {
-        transaction.getReactMountReady().enqueue(invokeComponentDidMountWithTimer, this);
+      if (__DEV__ && debugID !== 0) {
+        transaction.getReactMountReady().enqueue(() => {
+          measureLifeCyclePerf(
+            () => inst.componentDidMount(),
+            debugID,
+            'componentDidMount'
+          );
+        }, null);
```

Comment:
That’s callback context which is irrelevant since I use arrow anyway. I’m worried saying `/* context */` there will make it even more confusing because React overloads this term. Maybe I could get away with not passing it at all.